tests/unit/test_scraper.py

```
@@ -329,178 +494,36 @@ def test_performance_tracker_with_failures():
-def test_scrape_url_performance_tracking(mock_config, mock_response):
-    """スクレイピング関数のパフォーマンス計測テスト"""
+def test_scrape_url_validation_error_handling(mock_config, mock_response):
+    """検証エラーのハンドリングテスト"""
+    mock_response.text = """
+    <div class="tr">
+        <h2>Invalid Model</h2>
+        <div class="td2wrap">Invalid Price</div>
+    </div>
+    """
     with patch('requests.get', return_value=mock_response):
         prices = scrape_url('https://example.com/url1', mock_config)
```

Ruff (0.8.2)

Undefined name `scrape_url`

(F821)